hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieRealtimeRecordReaderUtils.java

```diff
@@ -306,7 +308,10 @@ public static Schema addPartitionFields(Schema schema, List<String> partitioning
   }
 
   public static HoodieFileReader getBaseFileReader(Path path, JobConf conf) throws IOException {
-    return HoodieFileReaderFactory.getReaderFactory(HoodieRecord.HoodieRecordType.AVRO).getFileReader(conf, path);
+    HoodieConfig hoodieConfig = new HoodieConfig();
+    hoodieConfig.setValue(USE_BUILT_IN_HFILE_READER,
+        Boolean.toString(ConfigUtils.getBooleanWithAltKeys(conf, USE_BUILT_IN_HFILE_READER)));
```

Comment:
instead of build in, can we say "NATIVE"

Reply:
Fixed.